### PR TITLE
Add Stock Trading

### PR DESCRIPTION
**Pull request by shi-rudo**
https://github.com/shi-rudo/awesome-stock-trading

List about valuable tools and educational resources about stock trading.

Commented pull requests: 
https://github.com/sindresorhus/awesome/pull/2391
https://github.com/sindresorhus/awesome/pull/2734
https://github.com/sindresorhus/awesome/pull/2692

## Requirements for your pull request

- ✅ Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. 
- ✅ **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- ✅ **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
- ✅ You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- ✅ This pull request has a title in the format `Add Name of List`. It should not contain the word `Awesome`.
- ✅ Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
- ✅ Your entry should be added at the bottom of the appropriate category.
- ✅ The title of your entry should be title-cased and the URL to your list should end in `#readme`.
- ✅ No blockchain-related lists.
- ✅ The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- ✅ **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- ✅ Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- ✅ The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- ✅ **Includes a succinct description of the project/theme at the top of the readme.
- ✅ It's the result of hard work and the best I could possibly produce.
- ✅ The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
- ✅ The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
- ✅ Non-generated Markdown file in a GitHub repo.
- ✅ The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- ✅ Not a duplicate. Please search for existing submissions.
- ✅ Only has awesome items. Awesome lists are curations of the best, not everything.
- ✅ Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- ✅ Includes a project logo/illustration whenever possible.
- ✅ Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- ✅ Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
- ✅ Has a Table of Contents section.
- ✅ Has an appropriate license.
- ✅ Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
- ✅ All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- ✅ Has consistent formatting and proper spelling/grammar.
- ✅ Does not use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- ✅ Does not include a CI (e.g. GitHub Actions) badge.<br>You can still use a CI for linting, but the badge has no value in the readme.
- ✅ Does not include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

### CODE REVIEW

**Comment by shi-rudo** [comment]
unicorn

**Comment by hyperupcall** [comment]
Awesome Stock Trading looks pretty good so far - I think there can be a few areas of improvement:

- Under the [books](https://github.com/shi-rudo/awesome-stock-trading#books) section, the descriptions for each book is kind of long. I think it would be easier to follow (and scroll) if each description was made more succinct (not autogenerating the description with AI).
- The repository description is kind of long, shorting it would make it more clear at a glance
- In general, not using AI to autogenerate descriptions, and writing them on your own to be more succinct, clear, and less obtuse
- It might be a good idea to link youtube channels with videos about stock market trading, kinds of trades one can make, strategies, etc.

**Comment by shi-rudo** [comment]
> Awesome Stock Trading looks pretty good so far - I think there can be a few areas of improvement:
> 
> * Under the [books](https://github.com/shi-rudo/awesome-stock-trading#books) section, the descriptions for each book is kind of long. I think it would be easier to follow (and scroll) if each description was made more succinct (not autogenerating the description with AI).
> * The repository description is kind of long, shorting it would make it more clear at a glance
> * In general, not using AI to autogenerate descriptions, and writing them on your own to be more succinct, clear, and less obtuse
> * It might be a good idea to link youtube channels with videos about stock market trading, kinds of trades one can make, strategies, etc.

Thank you. I took your suggestions to heart and agree with most of what you said. 

As for YouTube, I have to confess that I hardly use it, I'm more of a book guy. But I will look in the next few weeks to see if there is any good and substantial material.

**Comment by aabouzaid** [comment]
@shi-rudo According to the guides, the link and description are separated by a dash. The [Most Important Stock Exchanges](https://github.com/shi-rudo/awesome-stock-trading#most-important-stock-exchanges) section doesn't follow that.

**Comment by kfriede** [comment]
I'd recommend adding a space between your heading and the awesome badge

**Review by ahmed953 — APPROVED** [review]

**Comment by bdalalhaljwysm** [comment]
def fibonacci(n)
  return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end

**Review by Khaidahmedparray — APPROVED** [review]
[]()ghu44//yr6gg-&#//

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/shi-rudo/awesome-stock-trading
✖ Linting

  README.md:92:61
  ✖  92:61  List item description must start with valid casing  remark-lint:awesome-list-item
  ✖  93:64  List item description must start with valid casing  remark-lint:awesome-list-item

  2 errors
```

File: readme.md [hunk 1]
```diff
@@ -879,6 +879,7 @@
 - [Quarto](https://github.com/mcanouil/awesome-quarto#readme) - Scientific and technical open-source publishing system built on Pandoc.
 - [ChatGPT](https://github.com/sindresorhus/awesome-chatgpt#readme) - Artificial intelligence chatbot developed by OpenAI.
 - [Whisper](https://github.com/sindresorhus/awesome-whisper#readme) - Open-source AI-powered speech recognition system developed by OpenAI.
+- [Stock Trading](https://github.com/shi-rudo/awesome-stock-trading#readme) - Tools and educational resources about stock trading.
```
Comment (sindresorhus) [inline readme.md:882]: > It should not describe the list itself.  - guidelines

**Comment by shi-rudo** [comment]
> ```
> ❯ awesome-lint https://github.com/shi-rudo/awesome-stock-trading
> ✖ Linting
> 
>   README.md:92:61
>   ✖  92:61  List item description must start with valid casing  remark-lint:awesome-list-item
>   ✖  93:64  List item description must start with valid casing  remark-lint:awesome-list-item
> 
>   2 errors
> ```

**my awesome-lint v1 does not detect those errors!**

```shell
 awesome-finance % npx awesome-lint                                                  
✔ Linting
```

I even checked the lines with position you mentioned. there's nothing wrong? what do I miss?

```markdown
## Stock Collections
- [Growth 250](https://marketsmith.investors.com/growth250/) - MarketSmith's Growth 250 is a curated list of high-potential stocks.
- [StockTwits Top 10](https://stocktwits.com/rankings/trending) - StockTwits' list of the top 10 trending stocks.
```

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1757678455507026166

**Pull request merged**